app/assets/stylesheets/table_custom_striped.scss

```diff
@@ -3,3 +3,17 @@
     background-color: $table-bg-accent;
   }
 }
+
+.centered-column { text-align: center; }
+
+.table-item-rack-location {
+  background-color: inherit !important;
+  margin-bottom: 0px !important;
+  tr:first-of-type { border-top: 0px; }
+  tr { border-top: 1px solid #ddd; }
+  td {
+    border-top: 0px !important;
+    padding: 3px 0px !important;
+    width: 33%;
+  }
+}
```

Comment:
I'm not sure where you're keeping your styles, so I put these in what seemed like the appropriate place. Also, I know there are a lot of `!important`s in there, which I do not love. It was the only way I know how to override the bootstrap table styles. @tekHudson Do you know of a better way?